sdk/lib/opentelemetry/sdk/instrumentation_scope.rb

```diff
@@ -7,7 +7,11 @@
 module OpenTelemetry
   module SDK
     # InstrumentationScope is a struct containing scope information for export.
-    InstrumentationScope = Struct.new(:name,
-                                      :version)
+    InstrumentationScope = Struct.new(
+      :name,
+      :version,
+      :schema_url,
+      :attributes
+    )
```

Comment:
TODO: Look up docs for why I added `schema_url` and `attributes` to `InstrumentationScope` :thinking: